homeassistant/components/androidtv/__init__.py

```diff
@@ -157,9 +160,29 @@ async def async_connect_androidtv(
     return aftv, None
 
 
+def _migrate_old_screencap_option(
+    hass: HomeAssistant, entry: AndroidTVConfigEntry
+) -> None:
```

Comment:
I believe you should be using `async_migrate_entry`.

See docs [here](https://developers.home-assistant.io/docs/config_entries_config_flow_handler#config-entry-migration)

Reply:
Maybe I'm wrong but I don't think this is required just to replace an entry option. Migration is to be used when all config entry structure is changed.

Reply:
See PS4's migrate entry. Only used for country code change

https://github.com/home-assistant/core/blob/dev/homeassistant/components/ps4/__init__.py#L91

Reply:
Yes, but country code is in `data`, not in `options`, and probably are some mandatory data for integration. I have no problem in implementing this, but I managed other `options` migration where this was not required, so I just wait for a confirmation if this is required or not.